homeassistant/components/ring/siren.py

```diff
@@ -37,18 +35,18 @@ async def async_setup_entry(
 class RingChimeSiren(RingEntity, SirenEntity):
     """Creates a siren to play the test chimes of a Chime device."""
 
-    _attr_available_tones = CHIME_TEST_SOUND_KINDS
+    _attr_available_tones = list(CHIME_TEST_SOUND_KINDS)
     _attr_supported_features = SirenEntityFeature.TURN_ON | SirenEntityFeature.TONES
     _attr_translation_key = "siren"
 
-    def __init__(self, device: RingGeneric, coordinator: RingDataCoordinator) -> None:
+    def __init__(self, device, coordinator: RingDataCoordinator) -> None:
         """Initialize a Ring Chime siren."""
         super().__init__(device, coordinator)
         # Entity class attributes
         self._attr_unique_id = f"{self._device.id}-siren"
 
     @exception_wrap
-    def turn_on(self, **kwargs: Any) -> None:
+    def turn_on(self, **kwargs):
```

Comment:
This one doesn't have to dissapear

Reply:
I've tried with just `Any` or `-> None` as well as both of them together and in all instances I get the mypy error:
```
homeassistant/components/ring/siren.py:53: error: "RingGeneric" has no attribute "test_sound"  [attr-defined]
Found 1 error in 1 file (checked 13 source files)
```

Reply:
Allright, lets merge it for now and improve the typing in the followup